specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+weight. A _font family model_ defines what attributes may be used to differentiate fonts within a family.
+The new _typographic font family model_ has many advantages over the two previous font family models used
+on Windows. Unfortunately, changing font family models creates opportunities for confusion and compatibility
+issues. The methods exposed by the `IDWriteFontSet4` interface implement a hybrid approach that offers the
+advantages of the typographic font family model while mitigating compatibility issues.
+
+The following sections compare the older font family models with the typographic font family model,
+explain compatibility challenges posed by changing font family models, and finally explains how those
+challenges can be overcome by using the `IDWriteFontSet4` methods.
+
+## RBIZ Font Family Model
+
+The font family model used by GDI is sometimes called the "four-font model" or "RBIZ" model. Each font
+family in this model typically has at most four fonts. The "RBIZ" label comes from the naming convention
+used for some font files, for example:
+
+File Name       | Font Style
+----------------|-------------
+verdana.ttf     | Regular
+verdanab.ttf    | Bold
+verdanai.ttf    | Italic
+verdanaz.ttf    | Bold Italic
+
+With GDI, the input parameters used to select a font are defined by the `LOGFONT` structure, which
+includes family name (`lfFaceName`), weight (`lfWeight`) and italic (`lfItalic`) fields.
+
+The `lfItalic` field is a BOOL so it can only be TRUE or FALSE. The `lfWeight` field can technically be
+any weight in the range `FW_THIN` (100) to `FW_BLACK` (900), but is typically either `FW_NORMAL` (400)
+or `FW_BOLD` (700). For historical reasons, fonts designed for the RBIZ model almost always have one of 
+these two weights.
+
+To understand why, it helps to think of fonts as evolving in an ecosystem that also includes
+applications. Popular application user interfaces include an italic button (to turn italic on and off)
+and a bold button (to toggle between normal and bold weight). This user interface design made sense at
+a time when computer font families rarely had more than two weights. However, a side effect was that
+subsequent fonts could only have two weights within an RBIZ family if they wanted to work with
+existing applications.
+
+Suppose you wanted to add a heavier "Black" weight to the Arial font family. Logically, this font is
+part of the Arial family, so you might expect to select it by setting `lfFaceName` to "Arial" and 
+`lfWeight` to `FW_BLACK`. However, there is no way for an application user to choose between three
+weights using a two-state bold button. The solution was to give the new font a different family name,
+so the user could select it by choosing "Arial Black" from the list of font families. Likewise, there
+is no way to choose from among different widths in the same font family if your only affordances are
```

Comment:
Who's the target audience? The term "affordance" might not be familiar to all readers. (Just recently I used the term and then had to reword when someone asked for clarification.) You might say, "UI controls" instead.

Reply:
It's easy to avoid the word affordance here.